rsts/deployment/security/security.rst

```diff
@@ -43,3 +43,18 @@ context for these.
 For example: in case of `Flyteadmin <https://github.com/flyteorg/flyte/blob/master/charts/flyte/templates/admin/deployment.yaml>`__
 the init container of check-db-ready that runs postgres-provided docker image is not able to resolve the host for the checks and fails. This is mostly due to no read
 permissions on etc/hosts file. Only the check-db-ready container is run using the root user and which we will plan to fix as well.
+
+
+************
+OAuth
+************
+Flytectl needs a CA-certified SSL cert for OAuth to work. If a self-signed certificate is used, the following error is raised.
+
+.. code-block::
+    
+    certificate is not standards compliant.
+
```

Comment:
Flytectl requires CA-certified SSL cert for OAuth to work. Using a self-signed certificate throws the error: